cardano-config/src/Cardano/Config/Logging.hs

```diff
@@ -110,34 +137,46 @@ type LoggingCardanoFeature = CardanoFeatureInit
 
 -- | CLI specific data structure.
 data LoggingCLIArguments = LoggingCLIArguments
-  { logConfigFile :: !FilePath
+  { logConfigFile :: !(Maybe FilePath)
   , minSeverity   :: !Severity
   }
 
+data LoggingFlag = LoggingEnabled | LoggingDisabled
+  deriving (Eq, Show)
+
+-- | Interpret main logging CLI controls into a tuple of:
+--   - a designation of whether logging was disabled,
+--   - a valid 'LoggingConfiguration' (still necessary, even if logging was disabled)
+loggingCLIConfiguration :: LoggingCLIArguments -> IO (LoggingFlag, LoggingConfiguration)
+loggingCLIConfiguration (LoggingCLIArguments Nothing _) =
```

Comment:
I see a slight problem with this pattern matching. What if we add a field to LoggingCLIArguments ?
:-) you know, in prep..

Reply:
Very good point, fixed!